### ISSUE TITLE
Provide 32 bit binary

### ISSUE THREAD

**Issue by khurshid-alam** [iabem97/komorebi#54]
It would be nice if we can have 32 bit binary as well.

**Comment by khurshid-alam** [comment]
@iabem97 ?

**Comment by tesquin** [comment]
Yes, I agree. I went to the Releases page before I realised the .deb file was for 64-bit. Even if some of our machines are dog-slow with Ubuntu 18.xx, those of us who own i386 or i686 machines should not be locked out from using this utility.

S Wright (K/Ubuntu user since 12.04 -- though I skipped several versions until Unity was taken out of GNOME.)
